.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
```

Comment:
> I removed any configuration around semantic commit titles in the hope that it'll auto-detect based on the Git history. If it doesn't, I suspect we'd want to report that as a bug to Renovate.

Read the [Semantic Commit messages](https://docs.renovatebot.com/semantic-commits/) docs to learn more. But I'll quote the relevant part here:

> # Semantic Commit messages
> 
> Renovate looks at the last 10 commit messages in the base branch to decide if the repository uses semantic commits.
> If there are semantic commits, Renovate uses the [conventional-commits-detector](https://github.com/conventional-changelog/conventional-commits-detector) to decide what convention the commit messages follow.
> 
> Renovate only finds Angular-style conventional commits, it ignores other commit conventions.
> 
> When Renovate finds Angular-style commits, Renovate creates commit messages and PR titles like this:
> 
> - chore(deps): update eslint to v7.30.0
> 
> By default, Renovate uses the `chore` prefix.
> 
> If you extend from `config:recommended` then Renovate:
> 
> - still defaults to the `chore` prefix
> - uses the `fix` prefix for npm production dependencies
> - uses the `chore` prefix for npm development dependencies (`devDependencies`)

You're extending from `config:recommended` so, you'll get the behavior in the last part of the quote. 😉

Reply:
Cool, thanks, I wasn't sure that this was connected up to how we extended the config. I'll remove it as ESLint actually uses `chore: ...`, not `chore(deps): ...`.

Note that I didn't set the systems up and don't have ownership over it - this is just a proposal.

Reply:
Actually, wait, re-reading: are you saying that it will always be `chore(deps)`, not `chore`? ESLint uses `chore`, not `chore(deps)`. So if it's always prefix(scope) and not just prefix then we'd want to manually override here.

Reply:
## How I understand Renovate's docs for Commit Message

The way I'm reading our docs: if you don't use any presets, then Renovate uses `chore`. But if you use `config:recommended` then you still get `chore` for most things, with one exception:

- `fix` for npm production deps
- keep `chore` for npm development deps

## Why `config:recommended` appears to do the above behavior

`config:recommended` extends the `":semanticPrefixFixDepsChoreOthers",` preset which expands to this:

```json
{
  "packageRules": [
    {
      "matchPackagePatterns": [
        "*"
      ],
      "semanticCommitType": "chore"
    },
    {
      "matchDepTypes": [
        "dependencies",
        "require"
      ],
      "semanticCommitType": "fix"
    },
    {
      "matchDatasources": [
        "maven"
      ],
      "matchDepTypes": [
        "compile",
        "provided",
        "runtime",
        "system",
        "import",
        "parent"
      ],
      "semanticCommitType": "fix"
    }
  ]
}
```

## Error in the Renovate docs?

I'm getting confused here... [Renovate's Semantic Commit docs](https://docs.renovatebot.com/semantic-commits/#semantic-commit-messages) say:

> By default, Renovate uses the `chore` prefix.
> 
> If you extend from `config:recommended` then Renovate:
> 
> - still defaults to the `chore` prefix
> - uses the `fix` prefix for npm production dependencies
> - uses the `chore` prefix for npm development dependencies (`devDependencies`)

But the ``":semanticPrefixFixDepsChoreOthers"` preset seems to do something different. IT's a bit tricky for me to read.

### Question for Renovate maintainer

@rarkins should I change the Semantic Commit docs page, and if yes, how?

## Disabling the Semantic Commit scope (the `(chore)` part)

Quote from the [Renovate docs, Changing the Semantic Commit scope](https://docs.renovatebot.com/semantic-commits/#changing-the-semantic-commit-scope):

> ## Changing the Semantic Commit scope
> 
> ... snip ...
> 
> To _remove_ the semantic commit scope, so Renovate uses `chore:` instead of `chore(deps):`, add the `":semanticCommitScopeDisabled"` preset to your `extends` array:
> 
> ```json
> {
>   "extends": [":semanticCommitScopeDisabled"]
> }
> ```
>

Reply:
Added `":semanticCommitScopeDisabled"` to `extends`, thanks!